cvat-core/src/api-token.ts

```diff
@@ -0,0 +1,137 @@
+// Copyright (C) CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import PluginRegistry from './plugins';
+import serverProxy from './server-proxy';
+import { SerializedApiTokenData } from './server-response-types';
+import { APIApiTokenSaveFields, ApiTokenSaveFields } from './server-request-types';
+import { fieldsToSnakeCase } from './common';
+
+export default class ApiToken {
+    #id: number;
+    #name: string;
+    #createdDate: string;
+    #updatedDate: string;
+    #expiryDate: string | null;
+    #lastUsedDate: string | null;
+    #readOnly: boolean;
+    #owner: number;
+    #value?: string;
```

Comment:
Until initial create, some fields might be `undefined`.  E.g. see how you initiate the object in action creator.

So, some declared types seem not to be valid

Reply:
True, the token can be initialized with name, expiry and a readOnly. Updated